docs/kubernetes/deploy-to-google-cloud-from-ci.md

```diff
@@ -0,0 +1,113 @@
+| FIRST_DOMAIN_HOSTNAME       | domain hostname for first domain - without http://
+| SECOND_DOMAIN_HOSTNAME      | domain hostname for second domain - without http://
+| FIRST_DOMAIN_SSL_DIRECTORY  | path to SSL certificates for first domain
+| SECOND_DOMAIN_SSL_DIRECTORY | path to SSL certificates for second domain
+| PROJECT_ID                  | project id of your google project
+| GIT_COMMIT                  | commit hash to build your images uniquely
+| WORKSPACE                   | project root dir
+
+#### Mounts
+
+##### Docker Socket
+Docker socket is used to build and push image of php-fpm
+```
+-v /var/run/docker.sock:/var/run/docker.sock \
+```
+
+##### Terraform state
+Terraform state mounted locally to be able to apply changes or destroy terraform infrastructure.
+
+```
+-v ~/google-cloud/.terraform/tfstate:/tmp/infrastructure/google-cloud/tfstate \
+```
+
+##### SSL Certificates
+SSL certificates for secured protocol, mount these into folder of your choice that you choose setting environment variable.
+
+```
+-v ~/path/to/certificates-1:$FIRST_DOMAIN_SSL_DIRECTORY \
+-v ~/path/to/certificates-2:$SECOND_DOMAIN_SSL_DIRECTORY \
+```
+
+##### Google Account Service
+Mount your service-account.json obtained from google-cloud
+
+```
+-v ~/google-cloud/service-account.json:/tmp/infrastructure/google-cloud/service-account.json \
+```
+#### Usage
+Use Kubernetes buildpack with environment variable set:
+
+```bash
+docker run \
+    -v $WORKSPACE:/tmp \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -v ~/google-cloud/.terraform/tfstate:/tmp/infrastructure/google-cloud/tfstate \
+    -v ~/google-cloud/service-account.json:/tmp/infrastructure/google-cloud/service-account.json \
+    -v ~/path/to/certificates-1:$FIRST_DOMAIN_SSL_DIRECTORY \
+    -v ~/path/to/certificates-2:$SECOND_DOMAIN_SSL_DIRECTORY \
+    -e DOCKER_USERNAME \
+    -e DOCKER_PASSWORD \
+    -e GIT_COMMIT \
+    -e FIRST_DOMAIN_HOSTNAME \
+    -e SECOND_DOMAIN_HOSTNAME \
+    -e PROJECT_ID \
+    -e FIRST_DOMAIN_SSL_DIRECTORY \
+    -e SECOND_DOMAIN_SSL_DIRECTORY \
+    --rm \
+    shopsys/kubernetes-buildpack:0.2.0 \
+    .ci/deploy-to-google-cloud.sh
+```
+
+### Destroy Google Cloud Infrastructure
+Script used for deleting infrastructure on google. Script does not need any variable just a mount of Terraform `tfstate` file.
+
+[destroy-google-cloud-infrastructure.sh](/project-base/.ci/destroy-google-cloud-infrastructure.sh)
+
+#### Environment variables
+| Variable name          | Description
+| -------------          |-------------
+| PROJECT_ID             | project id of your google project
+
+##### Docker Socket
+Docker socket is used to build and push image of php-fpm
+```
+-v /var/run/docker.sock:/var/run/docker.sock \
+```
+
+##### Terraform state
+Terraform state mounted locally to be able to apply changes or destroy terraform infrastructure.
+
+```
+-v ~/google-cloud/.terraform/tfstate:/tmp/infrastructure/google-cloud/tfstate \
+```
+
+#### Usage
```

Comment:
Again, I would add a description when to use it a what will happen after the script is run